CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Compatibility:
 * Implement `MatchData#{byteoffset,deconstruct,deconstruct_keys}` from Ruby 3.2 (#3039, @rwstauner).
 * Add `Integer#ceildiv` method (#3039, @simonlevasseur, @nirvdrum).
 * Implement `Class#attached_object` method (#3039, @andrykonchin).
+* Implement `Time#deconstruct_keys` from Ruby 3.2 (@rwstauner).
```

Comment:
```suggestion
* Implement `Time#deconstruct_keys` from Ruby 3.2 (#3039, @rwstauner).
```

Reply:
Sorry, I misunderstood thinking this was an internal id.  Now that I realize it comes from the github issue that inspires the PR I'll try to put it in correctly the first time :)